lib/watcher/heart-beat.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+exports.worker = function(handle) {
+  var _heartBeatInterval = 15 * 1000;
+  setInterval(function() {
+    handle.send({
+      cmd: 'metrics',
+      record: JSON.stringify({dummyRecord: ''}),
+    });
+  }, _heartBeatInterval);
+};
```

Comment:
@sam-github : I do have the heartbeat generator. This file is the heart beat generator